src/devices_models/devices/hydro_generation.jl

```diff
@@ -235,6 +255,128 @@ function activepower_constraints!(psi_container::PSIContainer,
     return
 end
 
+######################## output constraints without Time Series ############################
+function _get_inflow_time_series(psi_container::PSIContainer,
```

Comment:
This feels like we should be able to do it in the `_get_time_series` function. But I think that's a separate PR.